tachyon/c/math/polynomials/constants.h

```diff
@@ -8,7 +8,7 @@ namespace tachyon::c::math {
 
 // NOTE(chokobole): We set |kMaxDegree| to |SIZE_MAX| on purpose to avoid
```

Comment:
```Suggestion
// NOTE(chokobole): We set |kMaxDegree| to |SIZE_MAX| - 1 on purpose to avoid
```

Reply:
I remember seeing this comment in multiple files. Is this the only one we need to change? If not, then is it not better to standardize the same named variables across files?

Reply:
Yes I changed [here](https://github.com/kroma-network/tachyon/pull/277/commits/146ad120c168b7b37faf0cd9bdd658d7b070e4dd)